assets/src/domain/eventEditor/ui/datetimes/datesList/addNew/ContentWrapper.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { __ } from '@wordpress/i18n';
+import { FormRenderProps } from 'react-final-form';
+
+import { withContext as withTAMContext } from '@edtrUI/ticketAssignmentsManager/context';
+import ModalBody from './ModalBody';
+import { Datetime } from '@edtrServices/apollo';
+
+/**
+ * This component is inside RFF context, so we can use all of RFF features.
+ */
+const ContentWrapper: React.FC<FormRenderProps> = (props) => {
+	const { values } = props.form.getState();
```

Comment:
let's add optional chaining, it scares me a little bit

Reply:
Those props are guaranteed from RFF. If that fails, there is no purpose of the whole form.